README.md

```diff
@@ -82,6 +82,45 @@ vm.$bindAsObject('user', myFirebaseRef.child('user'))
 vm.$bindAsArray('items', myFirebaseRef.child('items').limitToLast(25))
 ```
 
+To save user-input to your Firebase database, simply push the data onto `this.$firebaseRefs.messages` (instead of `this.messages`) within a Vue method to automatically sync with Firebase.
```

Comment:
I'll rather use `items` instead of `messages` to match previous examples. Also, it's a more generic term